test/logs/ARM_MUSCA_S1/CRYPTO.log

```diff
@@ -0,0 +1,6 @@
+Crypto Suite Report
```

Comment:
Logs added to `test/logs/ARM_MUSCA_S1/*.logs` , Are they in sync with (TF-M V 1.1)
**TF-M (Secure) + TF-M (Non-Secure) == TF-M (Secure) + Mbed OS (Non-secure)** passed results?

Also, I don't see any changes done to https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/tfm_ns_import.yaml, some files are required to be imported here, please refer to `ARM_MUSCA_B1` example.

Reply:
They are in sync with TF-Mv1.1 with the exception of the CRYPTO suite. I am using https://github.com/ARMmbed/trusted-firmware-m/pull/7 , and it contains crypto related changes which are not part of TF-Mv1.1.

I am using the hardware crypto accelerator and some of the tests fails, because MD5 is not supported. But is it a known issue and documented.